npm_and_yarn/spec/dependabot/npm_and_yarn/file_updater/yarn_lockfile_updater_spec.rb

```diff
@@ -163,7 +163,7 @@
       let(:files) { project_dependency_files("yarn/simple_with_registry_that_times_out") }
 
       it "raises a helpful error" do
-        pending("This test is extremely slow (1m45s) so only run locally. TODO: stub a custom timeout value.")
```

Comment:
This is not only a slow test, it's also a failing test, otherwise `pending()` wouldn't have worked:
* #6778